### PR TITLE
[now-cli] Fix inspector url with custom domain suffix

### PR DESCRIPTION
**Pull request by lucleray**
When the user is using custom domain suffix, the inspector url is a bit different:
```
https://zeit.co/{orgSlug}/{projectName}/{deploymentShortId}/{suffix}
```

This PR adds the necessary logic when constructing the inspector url.

cc @TooTallNate

### CODE REVIEW

File: packages/now-cli/src/util/deploy/process-deployment.ts [hunk 1]
```diff
@@ -20,15 +20,24 @@ function printInspectUrl(
...
-  const inspectUrl = `https://zeit.co/${orgName}/${projectName}/${deploymentShortId}`;
+  const url = deploymentUrl.replace('https://', '');
+
+  // example urls:
+  // lucim-fyulaijvg.now.sh
+  // s-66p6vb23x.n8.io (custom domain suffix)
+  const [sub, ...p] = url.split('.');
+  const apex = p.join('.');
+
+  const q = sub.split('-').reverse();
```
Comment (lucleray) [inline packages/now-cli/src/util/deploy/process-deployment.ts]:
```suggestion
  const q = sub.split('-');
```

File: packages/now-cli/src/util/deploy/process-deployment.ts [hunk 2]
```diff
@@ -20,15 +20,24 @@ function printInspectUrl(
...
-    .split('-');
-  const deploymentShortId = urlParts.pop();
-  const projectName = urlParts.join('-');
-  const inspectUrl = `https://zeit.co/${orgName}/${projectName}/${deploymentShortId}`;
+  const url = deploymentUrl.replace('https://', '');
+
+  // example urls:
+  // lucim-fyulaijvg.now.sh
+  // s-66p6vb23x.n8.io (custom domain suffix)
+  const [sub, ...p] = url.split('.');
```
Comment (TooTallNate) [inline packages/now-cli/src/util/deploy/process-deployment.ts:31]: You should use the `psl` module to parse this (`now.sh` is even [whitelisted](https://github.com/lupomontero/psl/blob/9775982e06068afbefad9dfb9d025b9d84bf1a9e/data/rules.json#L8805) as a public suffix):

```js
> psl.parse('asdfasdf.now.sh').tld
'now.sh'
```

**Review by TooTallNate — CHANGES_REQUESTED** [review]

File: packages/now-cli/src/util/deploy/process-deployment.ts (hunk 2 above)
Comment (lucleray) [reply packages/now-cli/src/util/deploy/process-deployment.ts:31]: That's great! I don't think this makes it easier to detect custom suffixes though 🤔

Comment (lucleray) [reply packages/now-cli/src/util/deploy/process-deployment.ts:31]: On second thought, the current code is more correct than what we can achieve with `psl` (for example, consider about the custom domain suffix `a.b.com`). I'd prefer to keep it as is.

Comment (TooTallNate) [reply packages/now-cli/src/util/deploy/process-deployment.ts:31]: I don't think we support subdomains for custom suffix, but ok, the current code LGTM anyways.

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**